docs/releases/5.0.txt

```diff
@@ -142,7 +142,11 @@ to create a field that is always computed from other fields. For example::
 
     class Square(models.Model):
         side = models.IntegerField()
-        area = models.GeneratedField(expression=F("side") * F("side"), db_persist=True)
+        area = models.GeneratedField(
+            expression=F("side") * F("side"),
+            output_field=models.IntegerField(),
```

Comment:
```suggestion
            output_field=models.BigIntegerField(),
```
It's important in this example to demonstrate how to correctly choose the output field.

Reply:
I don't think we should change to `BigIntegerField()` in all examples.

Reply:
This is the first example of the use of the `GeneratedField` that new users will see, and I think it is essential that it has a correct `output_field`, because, as has been pointed out to me, users will copy and paste that example without asking too many questions.
The same goes for fields defined in tests, because many developers, myself included, read Django code, trusting that it is correct.
What are your reasons for making these changes?

Reply:
I updated release notes.

> What are your reasons for making these changes?

I could ask the same :wink: 

> The same goes for fields defined in tests, because many developers, myself included, read Django code, trusting that it is correct.

I don't see anything incorrect in test examples. In most cases you will never jump into `BigIntegerField()` when adding two integers, it's not worth wasting storage space.

Reply:
> What are your reasons for not making these changes?

I'd argue that using the largest data type is not a good practice. You need to know your (expected) data. Tests do not use integers large enough to jump to the `BigIntegerField`, so there is no need to use them.

Reply:
You are always too quick for me to respond to comments and close suggestions. :)

I understand your reasoning, but IMO you underestimate the number of developers who read and copy Django test code.